native/src/main/java/io/ballerina/stdlib/ftp/util/FtpUtil.java

```diff
@@ -133,19 +143,22 @@ private static String normalizeFtpPath(String rawPath) {
         return "/" + rawPath;
     }
 
-    public static Map<String, String> getAuthMap(BMap config) {
+    public static Map<String, String> getAuthMap(BMap config, String protocol) {
```

Comment:
Would this be a breaking change? It is very unlikely that others will use this.

Reply:
Yes, I agree. It is unlikely to be used in other packages. It is an internal util function. Looks like this was made public for code organisation purposes.

Reply:
Overload method introduced to make it non breaking change.